### PR TITLE
Postgresql 14

### PR DESCRIPTION
**Pull request by CyberDem0n**
- removed PostgreSQL v9.5
- removed a couple of extensions installed from pgxn (they fail to compile on 11+ anyway)

### CODE REVIEW

File: postgres-appliance/Dockerfile [hunk 1]
```diff
@@ -218,33 +215,41 @@ RUN export DEBIAN_FRONTEND=noninteractive \
...
-        done; \
+        done \
+        && PACKAGES="partman repack"; \
     fi \
 \
     # build and install missing packages
-    && for pkg in pgextwlist; do \
-        apt-get source postgresql-$PGVERSION-${pkg} \
+    && for pkg in pgextwlist $PACKAGES; do \
+        apt-get source postgresql-13-${pkg} \
```
Comment (FxKu) [inline postgres-appliance/Dockerfile:238]: not $PGVERSION anymore?

Comment (CyberDem0n) [reply postgres-appliance/Dockerfile:236]: Only temporary. There is no repack package available for v14.

**Comment by FxKu** [comment]
:+1:

**Comment by CyberDem0n** [comment]
:+1:

**Pull request merged**